fastlane/lib/fastlane/fastlane_folder.rb

```diff
@@ -13,16 +13,31 @@ def self.path
       return value
     end
 
+    def self.fastfile_path
+      return nil if path.nil?
+      File.join(path, "Fastfile")
+    end
+
     # Does a fastlane configuration already exist?
     def self.setup?
       return false unless path
-      File.exist?(File.join(path, "Fastfile"))
+      File.exist?(self.fastfile_path)
     end
 
     def self.create_folder!(path = nil)
       path = File.join(path || '.', FOLDER_NAME)
       FileUtils.mkdir_p(path)
       UI.success "Created new folder '#{path}'."
     end
+
+    # Returns an array of symbols for the available lanes for the Fastfile
+    # This doesn't actually use the Fastfile parser, but only
+    # the available lanes. This way it's much faster
+    # Use this only if performance is :key:
+    def self.available_lanes
+      return [] if fastfile_path.nil?
+      output = `cat '#{fastfile_path}' | grep "^\s*lane \:" | awk -F ':' '{print $2}' | awk -F ' ' '{print $1}'`
```

Comment:
This needs to be really fast, so I want the parsing to happen using a shell script and not Ruby. If it's Ruby and someone uses the `update_fastlane` action, it will be super slow.